doc/nrf/libraries/others/event_manager_profiler.rst

```diff
@@ -0,0 +1,116 @@
+.. _event_manager_profiler_tracer:
+
+Event Manager profiler tracer
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Event Manager profiler tracer is a library that enables profiling Event Manager events.
+It serves as a linking layer between :ref:`event_manager` and the :ref:`profiler` and allows to profile Event Manager event submissions and processings as the Profiler events.
+The Profiler allows you to observe the propagation of an profiler event in the system, view the data connected with the event, or create statistics.
+
+The events managed by the Event Manager are structured data types that are defined by the application and can contain additional data.
+The Event Manager profiler tracer expands each event data structure with the :c:struct:`profiler_info` structure, which holds information required by the Profiler.
```

Comment:
The Event Manager profiler tracer registers :c:struct:`profiler_info` as ``trace_data`` of event types.
The tracer uses this additional information to profile events.